lib/blocs/auth/auth_bloc.dart

```diff
@@ -39,9 +45,16 @@ class AuthBloc extends Bloc<AuthEvent, AuthState> {
       try {
         // hardcoded value
         final userProfile = await userRepository.getUserProfileById(
-          2,
+          1,
         ); // for id = 1 feature disabled, for id = 2 enabled
 
+        featureFlagBloc.add(
+          FeatureFlagEvent.initialize(
+            featureName: Feature.cartsTab.featureName,
+            identity: fs.Identity(identifier: userProfile.email),
+          ),
+        );
```

Comment:
You need to await for featureFlag to load. FeatureFlagBloc.stream.firstWhere((state) => state.status == FeatureFlagStatus.complete)